waku-sys/src/cmd.c

```diff
@@ -0,0 +1,13 @@
+
+/*
+  This file is needed to avoid errors like the following when linking the waku-sys lib crate:
+  <<undefined reference to `cmdCount'>>
+  and
+  <<undefined reference to `cmdLine'>>
+*/
```

Comment:
> This is very interesting! do you know from where are these `cmdCount` and `cmdLine` variables being referenced?

This is needed because we have the following defined in `.../waku-rust-bindings/waku-sys/vendor/vendor/nimbus-build-system/vendor/Nim/lib/std/cmdline.nim`:

```nim
elif not defined(createNimRtl) and
  not(defined(posix) and appType == "lib"):
  # On Posix, there is no portable way to get the command line from a DLL.
  var
    cmdCount {.importc: "cmdCount".}: cint
    cmdLine {.importc: "cmdLine".}: cstringArray
 ```